internal/debug/flags.go

```diff
@@ -110,17 +116,36 @@ func init() {
 	glogger = log.NewGlogHandler(ostream)
 }
 
+// RotatingFileHandler returns a RotatingFileHandler
+func RotatingFileHandler(logdir string) (log.Handler, error) {
```

Comment:
what do we need this function for?

Reply:
It was for reusing the Rotatingfilehandler.
I removed it in this iteration.